web/src/components/FeedbackSearch.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement, useState } from 'react'
```

Comment:
Not sure about naming here, I think it is rather a `SearchFeedback`.

Reply:
Yes, but this way it is close to the other Feedback-stuff in the components folder. Maybe we can rename to `FeedbackForSearch` if you are really unhappy with this.

Reply:
I think from the context it should be clear what it is, if there is FeedbackEvents for example then it makes sense.